src/transform-sdk/js/main.cc

```diff
@@ -331,7 +724,46 @@ initial_native_modules(qjs::runtime* runtime, qjs::value* user_callback) {
           }
           return {std::exchange(*user_callback, std::move(args.front()))};
       });
-    return runtime->add_module(std::move(mod));
+    auto sr_mod = qjs::module_builder("@redpanda-data/sr");
```

Comment:
This should be `@redpanda-data/transform-sdk-sr` in case we ever have our own SDKs for non-wasm stuff.

Reply:
fair

Reply:
done